install_process.go

```diff
@@ -53,15 +54,17 @@ func (ip YarnInstallProcess) ShouldRun(workingDir string, metadata map[string]in
 	ip.logger.Break()
 
 	buffer := bytes.NewBuffer(nil)
+	listArgs := []string{"config", "list", "--silent"}
+	ip.logger.Subprocess("Running 'yarn %s'", strings.Join(listArgs, " "))
 
 	err = ip.executable.Execute(pexec.Execution{
-		Args:   []string{"config", "list", "--silent"},
-		Stdout: buffer,
-		Stderr: buffer,
+		Args:   listArgs,
+		Stdout: io.MultiWriter(ip.logger.ActionWriter, buffer),
+		Stderr: io.MultiWriter(ip.logger.ActionWriter, buffer),
 		Dir:    workingDir,
 	})
 	if err != nil {
-		return true, "", fmt.Errorf("failed to execute yarn config output:\n%s\nerror: %s", buffer.String(), err)
+		return true, "", fmt.Errorf("failed to execute yarn config output:\nerror: %s", err)
```

Comment:
I don't think we need to stream this to the user. This command is only run so that we can see if the configuration of yarn has changed since the last build. Its an internal implementation detail and not something I would expect a user to need to see.

Reply:
Fixed